File: 30days/08/08.py
#!/usr/bin/env python

n = int(input())

mylist = {}
for i in range(0, n):
    entry = input().split()
    mylist[entry[0]] = entry[1]

keepgoing = True

while (keepgoing):
    entry = input()
    if entry is None:
        keepgoing = False
    if entry in mylist:
        print(entry + "=" + mylist[entry])
    else:
        print("Not found")

File: 30days/08/08.rb
#!/usr/bin/env ruby

n = gets.chomp.to_i

mylist = {}

(0...n).each do
    entry = gets.chomp.split
    mylist[entry[0]] = entry[1]
end

keepgoing = true

until keepgoing == true
    entry = gets.chomp
    keepgoing = false if entry.nil?
    if mylist.key? entry
        puts "#{entry}=#{mylist[entry]}"
    else
        puts "Not found"
    end
end
